content/administration/maintain/azure_oauth.rst

```diff
@@ -0,0 +1,212 @@
+=======================================================
+Connect Microsoft Outlook 365 to Odoo using Azure OAuth
+=======================================================
+
+Odoo is compatible with Microsoft's Azure OAuth for Microsoft 365. In order to send and receive
+secure emails from a custom domain, all that is required is to configure a few settings on the
+Azure platform and on the back end of the Odoo database. This configuration works with either a
+personal email address or an address created by a custom domain.
+
+.. tip::
+   For more information, visit `Microsoft's documentation <https://learn.microsoft.com/en-us/azure/
+   active-directory/develop/quickstart-register-app>`_ on setting up OAuth in Azure.
```

Comment:
- If the tip is to suggest opening a link, then it's rather a seealso
- In RST, you can go to another line wherever you can write a space. So not in the middle of a link ^^
- Remove the region/language part of your URLs whenever possible (here the `/en-us`). This way, the link opens correctly for everyone, according to their cached settings on Microsoft's website or localization. 
```suggestion
.. seealso::
   `Microsoft Learn: Register an application with the Microsoft identity platform
   <https://learn.microsoft.com/azure/active-directory/develop/quickstart-register-app>`_ 
```

Reply:
It's just a single comment here, not a full review

Reply:
Hi @everyone.

Just a quick note while doing routine checks.

We can safely break in the middle of a hyperlink URL address as long as it's not in a tag like `:ref:` `:doc:` etc.

From the [RST guidelines](https://www.odoo.com/documentation/15.0/contributing/documentation/rst_guidelines.html#start-a-new-line-before-the-100th-character):
```
You can safely break a line around the separators (-->) of menuselection markups and anywhere in a
hyperlink reference. For the doc, ref and download markups, this is only true for the label part of
the reference.
```
This is confirmed by running a quick `make html` and seeing if the hyperlink breaks (which it shouldn't 😉). I can provide a number of live examples of where we did this on previous docs, if needed.

I do like JCS suggestion here though on turning this into a `:seealso:` which also allows for more context on the link before clicking it. Vague "click here" links aren't actually that helpful 😅